src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/HtmlCSharp/CompletionHandlerTest.cs

```diff
@@ -110,7 +111,8 @@ public async Task HandleRequestAsync_HtmlProjection_InvokesHtmlLanguageServer()
 
             // Assert
             Assert.True(called);
-            var item = Assert.Single((CompletionItem[])result.Value);
```

Comment:
Why'd you have to change this?

Reply:
We're now returning `DateTime` and all the keywords in this case, so `Assert.Single` will fail.

Reply:
Oh you were asking specifically for HtmlProjections, yeah that's the bug you pointed out above, will fix.